src/bun.js/bindings/generated_classes.zig

```diff
@@ -6327,6 +6327,9 @@ pub const JSTCPSocket = struct {
         if (@TypeOf(TCPSocket.getRemoteAddress) != GetterType)
             @compileLog("Expected TCPSocket.getRemoteAddress to be a getter");
 
+        if (@TypeOf(TCPSocket.getRemotePort) != GetterType)
+            @compileLog("Expected TCPSocket.getRemotePort to be a getter");
+
```

Comment:
Do the other conditions have test coverage? If so likely will need it for this too.

Reply:
This is generated code